cmd/substation/fmt.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/google/go-jsonnet/formatter"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(fmtCmd)
+	fmtCmd.PersistentFlags().BoolP("write", "w", false, "write result to (source) file instead of stdout")
+	fmtCmd.PersistentFlags().BoolP("recursive", "R", false, "recursively format all files")
+}
+
+var fmtCmd = &cobra.Command{
+	Use:   "fmt [path]",
+	Short: "Format Jsonnet files",
```

Comment:
```suggestion
	Short: "format configs",
```

This is a style change for consistency with the other commands, which currently look like this.
```sh
Available Commands:
  demo        demo substation
  fmt         Format Jsonnet files
  test        test configs
```